docs/supported_expectations.md

```diff
@@ -23,6 +23,8 @@ toHaveSameSizeAs
 ```
 notToHaveAttribute
 toHaveAttribute
```

Comment:
Please mark old method as deprecated or remove them completely.

By the way, link to docs dir in README.md are broken.

Reply:
Interesting.. technically, the links should work starting with `/`, github docs specifically say those are relative to the repository root. Yet, they are not.

Maybe it's a special case for `/docs`, since that also houses github docs itself.

Reply:
Removed the attribute assertions from the expectations and verifier docs, and attempted a fix at the README... the README links now _should_ work, yet I am unable to test it since if I render it on my branch, it's relative to the `/tree/` url.